WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/data/db/WatchFaceArmDimensionsDao.kt

```diff
@@ -21,18 +21,17 @@ import androidx.room.Insert
 import androidx.room.OnConflictStrategy
 import androidx.room.Query
 import androidx.room.Update
-import kotlinx.coroutines.flow.Flow
 
 /**
  * Defines [WatchFaceArmDimensionsEntity] database operations.
  */
 @Dao
 interface WatchFaceArmDimensionsDao {
     @Query("SELECT * FROM watch_face_arm_dimensions_table ORDER BY id ASC")
-    fun getAll(): Flow<List<WatchFaceArmDimensionsEntity>>
+    suspend fun getAll(): List<WatchFaceArmDimensionsEntity>
 
-    @Query("SELECT * FROM watch_face_arm_dimensions_table WHERE id=(:id)")
-    fun get(id: String): Flow<WatchFaceArmDimensionsEntity>
+    @Query("SELECT * FROM watch_face_arm_dimensions_table WHERE id=(:id) LIMIT 1")
```

Comment:
Add a comment saying why you limit to 1

Reply:
I saw it in the [Sunflower sample](https://github.com/android/sunflower/blob/main/app/src/main/java/com/google/samples/apps/sunflower/data/GardenPlantingDao.kt#L34) and I saw it may [increase performance](https://stackoverflow.com/questions/455476/does-adding-limit-1-to-mysql-queries-make-them-faster-when-you-know-there-will#:~:text=If%20you%20want%20only%20one,definitely%20add%20a%20LIMIT%20clause.), but I guess probably not for my small DB.

I can take out for all or add comment saying it may improve performance, what do you recommend?

Reply:
I thought you're limiting it to 1 because you want to make sure you're not getting a list.

Reply:
Well, I was doing that too. :D 

I added a comment. WDYT?